README.md

```diff
@@ -1,2 +1,22 @@
 # comp110-worksheet-C
 Base repository for COMP110 worksheet C
+
+(a)go through two lists then see if the list are different sizes but have the same final number
```

Comment:
There is only one list -- the algorithm checks if it contains any duplicated elements.